src/main/scala/gemmini/Controller.scala

```diff
@@ -263,12 +266,29 @@ class GemminiModule[T <: Data: Arithmetic, U <: Data, V <: Data]
   }
   */
 
-  load_controller.io.cmd.valid := rob.io.issue.ld.valid
+  val load_cmd_ready = load_controller.map(_.io.cmd.ready)
+  rob.io.issue.ld.ready := load_cmd_ready.reduce(_||_)
+
+  // TODO: would this work fine? (want to send cmd to a single controller that is ready)
+  val req_ld = MuxCase(0.U, Seq.tabulate(num_data_controller){
+    i => load_controller(i).io.cmd.ready -> i.asUInt()
+  })
+  load_controller.zipWithIndex.foreach{case(d, i) =>
```

Comment:
We actually probably want to have a round-robin arbiter here, else the commands may end up in the queue in a single LoadController.

Reply:
Also need to consider the case where a CONFIG_CMD needs to broadcast to all LoadControllers

Reply:
Oh, yeah. You're right. Forgot about the config command

Reply:
okay just made a change for this